apps/api/src/app/portfolio/calculator/roai/portfolio-calculator-novn-buy-and-sell.spec.ts

```diff
@@ -145,19 +145,23 @@ describe('PortfolioCalculator', () => {
         valueWithCurrencyEffect: 0
       });
 
+      /**
+       * Closing price on 2022-03-07 is unknown,
+       * hence it uses the last unit price (2022-04-11): 87.8
+       */
       expect(portfolioSnapshot.historicalData[1]).toEqual({
         date: '2022-03-07',
         investmentValueWithCurrencyEffect: 151.6,
-        netPerformance: 0,
+        netPerformance: 24, // 2 * (87.8 - 75.8) = 24

```

Comment:
Why are `netPerformanceInPercentage` and `netPerformanceInPercentageWithCurrencyEffect` equal to `0` when `netPerformance` has a positive value?